docs/source/state-machine.rst

```diff
@@ -332,6 +332,19 @@ abort instead.
     actions that it is not safe to replay: staging a device, adding a
     monitor, or adding a subscription.
 
+.. caution::
+
+    This plan will exit the entire Python session immediately:
```

Comment:
This puts the RE in a paused state, "exit the entire Python session" to me reads as `sys.exit(0)` which is not what this is doing....

Reply:
That's a more precise statement.  Yet isn't the immediate _consequence_ that the Python process is terminated?

Reply:
I would expect that this would give you the IPython prompt back?